yarn.lock

```diff
@@ -3025,20 +3025,20 @@ __metadata:
   languageName: node
```

Comment:
`yarn.lock` was updated! It's showing up at the bottom of the changed files list, though:
![](https://user-images.githubusercontent.com/46334387/185900570-42f78d75-b8f3-497e-903f-7206f7bb9d5f.png)